mastermind-services/mongo/docker-compose.yml

```diff
@@ -35,9 +35,9 @@ services:
         # TODO: Avoid exposing the docker socket (security issue)
         - /var/run/docker.sock:/var/run/docker.sock
       environment:
-        - OVERLAY_NETWORK_NAME=${OVERLAY_NETWORK_NAME:-mongo_backend}
-        - MONGO_SERVICE_NAME=${STACK_NAME:-mongo}_mongo
-        - REPLICASET_NAME=${REPLICASET_NAME:-rs}
+        - OVERLAY_NETWORK_NAME=${OVERLAY_NETWORK_NAME}
```

Comment:
Why does comet use {{}} and this one ${} ?

Reply:
The mongo compose is actually not updated to work with the service manager right now, because it uses an external file. The compose files in the "catalog" here in general are temporary, they are mostly for demonstrating that services can be deployed. Eventually we'll make a proper catalog with the actual up to date compose files and accompaning mastermind configs.